### ISSUE TITLE
Robomongo 1.0 uses wrong config paths

### ISSUE THREAD

**Issue by hackel** [Studio3T/robomongo#1350]
I just noticed the existence of a .3T directory in my home directory.  This is not where config files belong, per XDG standards.  I noticed the following change:
```cpp
    // Change of path due to re-branding to 3T
    const auto CONFIG_FILE_1_0_0 = QString("%1/.3T/robomongo/1.0.0/robomongo.json").arg(QDir::homePath());
```
This is wrong.  There is *no* reason to include "3T branding" in a config file path.  ~/.config/robomongo is the correct location.  ~/.config/3t/robomongo would be slightly better, but still not ideal as the additional 3t directory is entirely redundant.

Likewise, the cache should be stored in ~/.cache/robomongo, not ~/.3T/robomongo/1.0.0/cache.

For more information, see https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
I'm sure similar equivalents exist for proprietary operating systems as well.

**Comment by revolter** [comment]
I highly agree with this too.

**Comment by simsekgokhan** [comment]
Hi @hackel,  thanks a lot for reporting this problem. 
I hope we will discuss and solve this problem soon.

**Comment by outreal** [comment]
I totally agree with @hackel. I understand that, having acquired robomongo, 3T wants to rebrand it, which is perfectly understandable, but please respect XDG standards in the process :wink:

**Comment by hackel** [comment]
Very disappointing to see this is still not fixed in "Mongo 3T" 1.1, even though attention was obviously given to the new config paths, as mentioned in: http://blog.robomongo.org/robomongo-is-robo-3t/#5

Config files must go into `/home/<user>/.config/robo-3t/robo3t.json` on Linux.

Littering users' config directories with files for old versions also doesn't make sense. If users want to run and old version simultaneously, there should be a --config parameter to specify an alternate configuration path.

**Comment by kuba-gaj** [comment]
2 years later and still no change :(
